### PR TITLE
product page added

### PR DESCRIPTION
**Pull request by yogesh-**
- removed css erros
- resolved api call and map issues

https://user-images.githubusercontent.com/7193961/160374300-03948c6d-993c-45f0-87da-747858dcc229.mp4

### CODE REVIEW

File: src/pages/products/products.jsx [hunk 1]
```diff
@@ -1,12 +1,152 @@
...
+  useEffect(() => {
+    loader();
+  }, []);
+
+  const loader = async () => {
+    try {
+      setLoad(true);
+      const response = await axios.get("/api/products/");
+      console.log(response.data.products);
+      console.log("hello");
```
Comment (tushargoyal812) [inline src/pages/products/products.jsx:20]: remove console

**Review by tushargoyal812 — COMMENTED** [review]
file structure is good
variables names are good.
code is short and clean.

File: src/pages/products/products.jsx (hunk 1 above)
Comment (yogesh-) [reply src/pages/products/products.jsx]: done

**Review by akash-1318 — COMMENTED** [review]
Clean code with good file structure and meaningful naming , you can continue merge from my end.

File: src/pages/products/products.jsx [hunk 2]
```diff
@@ -1,12 +1,150 @@
-import react from "react";
+import React, { useEffect, useState } from "react";
+import axios from "axios";
 import { Footer } from "../../components/footer/footer";
```
Comment (pariyar07) [inline src/pages/products/products.jsx:3]: try to use absolute imports. Looks clean and will be easy to manage if its a big project.

File: src/pages/products/products.jsx [hunk 3]
```diff
@@ -1,12 +1,150 @@
...
 import { Navbar } from "../../components/navbar/navbar";
+import "./products.css";
 
 export const Products = () => {
+  const [data, setData] = useState([]);
+  const [load, setLoad] = useState(false);
+
+  useEffect(() => {
+    loader();
```
Comment (pariyar07) [inline src/pages/products/products.jsx:12]: try to add this in a context.

**Comment by pariyar07** [comment]
Few suggestions. Other looks good.

**Pull request merged**